tests/e2e/create/create_test.go

```diff
@@ -247,23 +279,6 @@ var _ = Describe("Test 'create' command", Ordered, func() {
 		})
 	})
 
-	Context("Given 'modulectl create' command", func() {
-		var cmd createCmd
-		It("When invoked with '--config-file' using valid file", func() {
-			cmd = createCmd{
-				moduleConfigFile: minimalConfig,
-			}
-		})
-		It("Then the command should succeed", func() {
-			Expect(cmd.execute()).To(Succeed())
-
-			By("And no module template file should be generated")
-			currentDir, err := os.Getwd()
-			Expect(err).ToNot(HaveOccurred())
-			Expect(filesIn(currentDir)).Should(Not(ContainElement("template.yaml")))
-		})
-	})
-
```

Comment:
Previously we didn't write the ModuleTemplate when registry was empty. As we don#t support empty registry anymore and always write the ModuleTempalte this test is not needed anymore.